source/structured_data.cc

```diff
@@ -604,6 +607,174 @@ namespace aspect
     }
 
 
+
+    template <int dim>
+    void
+    StructuredDataLookup<dim>::load_netcdf(const std::string &filename)
+    {
+#ifndef ASPECT_WITH_NETCDF
+      (void)filename;
+      AssertThrow(false, ExcMessage("Loading NetCDF files is only supported if ASPECT is configured with the NetCDF library!"));
```

Comment:
Is it ASPECT or deal.II? (I forgot which project provides the netCDF bindings...)

Reply:
ASPECT.